pkg/alertmanager/multitenant.go

```diff
@@ -1025,6 +1084,33 @@ func (am *MultitenantAlertmanager) serveRequest(w http.ResponseWriter, req *http
 	http.Error(w, "the Alertmanager is not configured", http.StatusPreconditionFailed)
 }
 
+// startAlertmanager will start the Alertmanager for a tenant, using the fallback configuration if no config is found.
+func (am *MultitenantAlertmanager) startAlertmanager(ctx context.Context, userID string) (*Alertmanager, error) {
```

Comment:
I think this method can be combined with `alertmanagerFromFallbackConfig`, they are effectively doing the same thing. Probably keep the `startAlertmanager` name and make sure the correct errors are returned from the combined method.

Reply:
I didn't want to change the behavior of Mimir Alertmanager tenants, at least not yet. I was planning on doing that on a future PR, when applying strict initialization to all tenants.